scripts/cti

```diff
@@ -58,6 +63,7 @@ then
       echo "-W|--workspace <WORKSPACE>: Use custom workplace <WORKSPACE>"
       echo "-E|--env <ENV>: Set environment variable, ex. -E RUST_LOG=debug. Can be repeated, e.g. -E A=B -E C=D"
       echo "-R|--report file.json: Generate json report into file.json"
+      echo "--marker <MARKER>: Marker to identify purpose of this run. Must be set by automatic invocations like CI"
```

Comment:
Curious, what does it actually do behind the scene in CTI?

Reply:
For now it just logs with this marker, so we can understand what is going on